### ISSUE TITLE
cannot use zip dependencies for lxml in Python lambda

### ISSUE THREAD

**Issue by kniklas** [kniklas/finscrap#18]
TLDR: Following article must be followed, to overcome the problem: https://repost.aws/questions/QUwIZg6DlXTgKHjJyMIlG1dQ/unable-to-import-lxml-etree-on-aws-lambda

Few other sources with similar problems:
* https://repost.aws/knowledge-center/lambda-layer-simulated-docker
* https://gist.github.com/allen-munsch/ad8faf9c04b72aa8d0808fa8953bc639
* https://stackoverflow.com/questions/56818579/unable-to-import-lxml-etree-on-aws-lambda/67267811#67267811

The problem originates, from fact, that lxml package, installed in MacOS has Darwin compiled binaries
<img width="658" alt="Screenshot 2023-06-20 at 22 21 57" src="https://github.com/kniklas/finscrap/assets/5281192/5de0f966-8cd0-47ff-9b9d-a0fd9d84e839">

**Comment by kniklas** [comment]
As per the first link, potentially following could work, however `finscrap` package must be available in PyPI server.

Apply following:
```shell
$ docker run -v $(pwd):/outputs -it amazonlinux:latest
yum update -y
yum install python39 python-pip libxml2 libxslt
pip3 install lxml
cd home

# from host copy requirements-build.txt, XXXXXXX - check id of container
docker ps
docker cp requirements-build.txt XXXXXXXXXXXXX:/requirements.txt

# from docker container shell install dependencies
# pip3 install --target . -r requirements.txt #  OPTION?
# Note to use correct version from testPYPI!
pip3 install --target . -i https://test.pypi.org/simple/ finscrap==0.0.dev8

# copy from docker container files to packages folder - make sure correct local folder name is used!
docker cp -r 9a8509a8370b:/home/* ./packages/

# Zip dependencies
zip -r packages.zip packages/
zip packages.zip get_data.py funds-short.json

# Create lambda, make sure to use correct role
aws lambda create-function --function-name get-data2 --runtime python3.9 --role arn:aws:iam::${AWS_ACCOUNT}:role/lambda-apigateway-role --handler get_data.lambda_handler --zip-file fileb://packages.zip

**Comment by kniklas** [comment]
Closing as won't fix. Used docker image.
